cleanlab/rank.py

```diff
@@ -286,6 +291,12 @@ def get_label_quality_ensemble_scores(
             """
         )
 
+    # Raise a warning if user passed custom_weights array but did not choose weight_ensemble_members_by="custom"
+    if custom_weights is not None and weight_ensemble_members_by != "custom":
+        warnings.warn(
```

Comment:
ValueError instead of warning? It seems misleading to ignore custom weights if they were purposefully passed in.

Reply:
Thanks, good call. I made this change.

Initially, I chose warning because I thought the user could have accidentally passed a `custom_weights` array when choosing `"accuracy"` or `"uniform"` for `weight_ensemble_members_by`. But now that I think about it, this is unlikely and it makes more sense to raise a ValueError.